### DISCUSSION TITLE
How to classify edges of a Heterogenous Graph?

### DISCUSSION THREAD

**Discussion by avijit9** [pyg-team/pytorch_geometric#3605] [Q&A]
Hi,

I need a kickstart to classify the edges of a particular type of edges in a heterogeneous graph. Is converting the generic model from the homogeneous graph is a solution?

Any help would be appreciated.

**Comment by rusty1s** [comment]
Yes, I think so. You just need to add a final edge classifier on top, see https://github.com/pyg-team/pytorch_geometric/pull/3350
